Changelog.md

```diff
@@ -1,5 +1,168 @@
+
+1. Fix Joint Position Controller Behaviour Described in #1997
+    * [Pull request #2001](https://github.com/gazebosim/gz-sim/pull/2001)
+
+1. Include contact force, normal, and depth in contact message
+    * [Pull request #2050](https://github.com/gazebosim/gz-sim/pull/2050)
+
+1. Use sdf::Element::FindElement instead of GetElement in ApplyLinkWrench
+    * [Pull request #2052](https://github.com/gazebosim/gz-sim/pull/2052)
+
+1. Backport sensors system threading optimization changes
+    * [Pull request #2058](https://github.com/gazebosim/gz-sim/pull/2058)
+
+1. Apply Force and Torque GUI plugin
+    * [Pull request #2014](https://github.com/gazebosim/gz-sim/pull/2014)
+
+1. Adds a warning if the `Server` method of a `TestFixture` is called before `Finalize`
+    * [Pull request #2047](https://github.com/gazebosim/gz-sim/pull/2047)
+
+1. Support loading mesh by mesh name in `<mesh><uri>`
+    * [Pull request #2007](https://github.com/gazebosim/gz-sim/pull/2007)
+
+1. ComponentInspector: display PhysicsEnginePlugin
+    * [Pull request #2032](https://github.com/gazebosim/gz-sim/pull/2032)
+
+1. Send BlockOrbit false events only once from TransformControl plugin
+    * [Pull request #2030](https://github.com/gazebosim/gz-sim/pull/2030)
+
+1. Categorize tutorials list
+    * [Pull request #2028](https://github.com/gazebosim/gz-sim/pull/2028)
+
+1. Add time out to wait to avoid deadlock
+    * [Pull request #2025](https://github.com/gazebosim/gz-sim/pull/2025)
+
+1. Add optional binary relocatability
+    * [Pull request #1968](https://github.com/gazebosim/gz-sim/pull/1968)
+
+1. Several minor fixes
+    * [Pull request #2027](https://github.com/gazebosim/gz-sim/pull/2027)
+
+1. Protobuf: Do not require version 3 do support Protobuf 4.23.2 (23.2)
+    * [Pull request #2006](https://github.com/gazebosim/gz-sim/pull/2006)
+
+1. Support world joints (joints inside `<world>` tags)
+    * [Pull request #1949](https://github.com/gazebosim/gz-sim/pull/1949)
+
+1. Disable pybind11 on Windows by default
+    * [Pull request #2005](https://github.com/gazebosim/gz-sim/pull/2005)
+
+1. Port record topic fix
+    * [Pull request #2004](https://github.com/gazebosim/gz-sim/pull/2004)
+
+1. Allow re-attaching detached joint
+    * [Pull request #1687](https://github.com/gazebosim/gz-sim/pull/1687)
+
+1.  Enable GzWeb visualization of markers by republishing service requests on a topic
+    * [Pull request #1994](https://github.com/gazebosim/gz-sim/pull/1994)
+
+1. Support loading Projectors
+    * [Pull request #1979](https://github.com/gazebosim/gz-sim/pull/1979)
+
+1. Small fixes to gz headers
+    * [Pull request #1985](https://github.com/gazebosim/gz-sim/pull/1985)
+
+1. Speed up Resource Spawner load time by fetching model list asynchronously
+    * [Pull request #1962](https://github.com/gazebosim/gz-sim/pull/1962)
+
+1. Use `ignition::gazebo::` in class instantiation
+    * [Pull request #1967](https://github.com/gazebosim/gz-sim/pull/1967)
```

Comment:
this one is probably not as relevant for garden?

Reply:
Yup, removed in https://github.com/gazebosim/gz-sim/pull/2173/commits/ad6f7e80e007209572bbd0286fcf3836561a786e